.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: java
+sudo: false
+script: mvn clean verify
```

Comment:
we probably need an `-s ~/settings-odl.xml` with the settings-odl.xml here? 

Lemme set up a Travis on this repo right away...

Reply:
Yes, we'll need that if we have to pass it a custom settings.xml file.